packages/editor/src/services/ComponentShelfCategoriesState.ts

```diff
@@ -105,27 +103,24 @@ export const ComponentShelfCategoriesState = defineState({
         DirectionalLightComponent,
         HemisphereLightComponent
       ],
-      FX: [
-        LoopAnimationComponent,
-        ShadowComponent,
-        ParticleSystemComponent,
-        EnvmapComponent,
-        SDFComponent,
-        PostProcessingComponent,
-        ReflectionProbeComponent
-      ],
-      Scripting: [VisualScriptComponent],
+      FX: [LoopAnimationComponent, ShadowComponent, ParticleSystemComponent, EnvmapComponent, PostProcessingComponent],
+      Scripting: [],
```

Comment:
if its empty, why keep the scripting field at all? 
also have we removed the system Component as well?

Reply:
Keep it defined. It's not really needed but simplifies the reactor logic a tiny bit